app/github.js

```diff
@@ -50,7 +50,7 @@ export async function addLabel(token, login, repository, labelableId, labels) {
         ","
       )}
 
-Check that the label exists and is spelt right then try again.
+Check that [the label exists](https://github.com/${login}/${repository}/labels) and is spelt right then try again.
```

Comment:
can you also update https://github.com/lemeurherve/github-comment-ops/blob/patch-1/app/github.js#L134-L140 please?